pagopa/pn-frontend
# fixed mixpanel missed events - pn-12540

## Short description
Send address details on login and send event on "Personalizza recapiti" click

## List of changes proposed in this pull request
- Renamed SEND_ADD_SECONDARY_CONTACT into SEND_ADD_CUSTOMIZED_CONTACT
- Restored getDomicileInfo action

## How to test
Login with pf -> check that SEND_HAS_ADDRESSES is logged
Login with pf -> go to recapiti page -> add legal contact -> click on "Personalizza recapiti" -> check that SEND_ADD_CUSTOMIZED_CONTACT is logged
